server/src/instant/model/app.clj

```diff
@@ -273,6 +275,17 @@
                             WHERE a.id = ?::uuid"
                            new-creator-id id])))
 
+(defn clear-by-id!
+  "Deletes attrs, rules, and triples for the specified app_id"
+  ([params] (clear-by-id! aurora/conn-pool params))
+  ([conn {:keys [id]}]
+   (next-jdbc/with-transaction [tx-conn conn]
+     (attr-model/delete-by-app-id! tx-conn id)
+     (rule-model/delete-by-app-id! tx-conn {:app-id id}))))
```

Comment:
To get this to invalidate queries, you probably need to add:

```
(transaction-model/create! conn {:app-id id})
```

inside the transaction.